docs/CONTRIBUTING.md

```diff
@@ -35,24 +35,29 @@ kubectl-gadget-linux-amd64` or `make kubectl-gadget-darwin-amd64`.
 
 ### Building the gadget container image
 
+Inspektor Gadget provides two different container images:
+- gadget-bcc: Contains CO-RE and BCC gadgets
+- gadget-core: Containes only CO-RE gadgets
```

Comment:
What about non CO-RE gadgets that are not BCC (traceloop, network-policy advisor)?

Reply:
Added a comment clarifying what happens with them.